pkg/service/auth_context.go

```diff
@@ -249,3 +249,25 @@ func (authContext *AuthContext) GetResolvedMetadata() map[interface{}]interface{
 	}
 	return m
 }
+
+type authContextData struct {
+	Request *envoy_auth.AttributeContext `json:"context"`
+	Context map[string]interface{}       `json:"auth"`
+}
+
+func (authContext *AuthContext) ToData() interface{} {
```

Comment:
I'm not sure what this `ToData` func transforms to. Maybe we could define with a type the desired shape of Data?

Reply:
I couldn't find any better name for it. That is what it does, i.e. it extracts from the auth context only that data that matters.

Maybe we could name it "ToAuthorizationData", but I see how even this can be confusing.

Reply:
Renamed to `GetDataForAuthorization()`